cli/src/remote/index.js

```diff
@@ -239,11 +239,30 @@ export default class Remote implements Side {
         ifMatch: doc.remote._rev
       })
     } catch (err) {
+      // FIXME: Don't complain when file/dir is already trashed/deleted?
```

Comment:
I'd be better to do that, but it's not a big deal: the watcher will see that the file/dir was trashed/deleted and the file/dir will be marked as such in pouchdb